app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
+            break
+
+    filename = argv[index]
+
+    if "-d" in argv:
+        index2 = 0
+
+        for _ in range(len(argv) - 1):
+            index2 += 1
+            if argv[_: _ + 1] == ["-d"]:
+                break
+
+        directories = argv[index2:index - 1]
+else:
+    filename = "file.txt"
+
+    if "-d" in argv:
+        index = 0
+
+        for _ in range(len(argv) - 1):
+            index += 1
+            if argv[_: _ + 1] == ["-d"]:
+                break
+
+        directories = argv[index:len(argv)]
+
+file_data = [datetime.today().strftime('%y:%d:%m %H-%M-%S'), "\n"]
+
+while True:
+    line = input("Enter content line: ")
+    if line == "exit":
+        break
+    file_data.extend([line])
+
+for directory in directories:
+    try:
+        mkdir(directory)
+    except Exception:
+        pass
+    chdir(directory)
+
+with open(filename, "w") as f:
+    f.writelines(file_data)
```

Comment:
The task requirements state that `If file.txt already exists it should add content below`. Using the mode `"w"` in `open()` function will overwrite the file content each time we run the script. Instead we need to use mode `"a"`, that will append the file content at the end of the file.
And to separate the file content from each script run, we also need to add "\n\n" at the end of the file.
So that it'll look like:
```suggestion
with open(filename, "a") as f:
    f.writelines(file_data)
    f.write("\n\n")
```